lib/modules/versioning/pep440/index.ts

```diff
@@ -29,8 +29,8 @@ const {
 
 function isVersion(input: string | undefined | null): boolean {
   // @renovatebot/pep440 isn't strict null save
-  // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
-  return !!valid(input!);
+
+  return !!valid(input);
```

Comment:
@rarkins  i did `lint-fix` and this happened? or someone else did and its just a merge? i didnt change this. should i roll it back?

Reply:
@viceice do you know?

I guess it can likely be rolled back (assuming `yarn lint` does not error it) but I'm curious to see `lint-fix` making such a change

Reply:
i rolled it back